### ISSUE TITLE
Add cordova-plugins on plugin install?

### ISSUE THREAD

**Issue by Suven** [isleofcode/ember-cordova#195]
Hey!

I am writing a few wrappers around cordova-plugins in form of services to easier use them.

For some of them it makes sense to ship them via an ember-addon, however I would need to point out to the user that he still needs to install the cordova-plugin himself.

I would love if I could instead, just reference ember-cordova as dep and run `cordova plugin add foo --save` on install-time via my plugins index.js.

Is that already possible? I really would appreciate guidance for that.

**Comment by Suven** [comment]
Argh. Sorry, I just found out that you are already doing this for example [here](https://github.com/isleofcode/ember-cordova-keyboard/blob/master/blueprints/ember-cordova-keyboard/index.js).

Thanks a lot for your efforts!

**Comment by Suven** [comment]
I just released the first wrapper as [ember-cordova-storage](https://github.com/Suven/ember-cordova-storage) which reads/writes data either through localStorage or the native iOS/Android/Win Settings-Storages.

One thing I duplicated for that was your [lib/await-command.js](https://github.com/Suven/ember-cordova-storage/blob/master/lib/await-command.js). I hope thats okay? :/ If you wish I can stronger link you in source-code. Right now I referenced you in the readme.

That said, I still think ember-cordova would still make for a good general base. Maybe such an hook is also possible without needing third-party plugins to  write the same code over and over again. While technically quite similar, I think that this convinience is what made ionic that strong in the first place.
